static/js/user/page.tsx

```diff
@@ -76,7 +76,7 @@ export function Page(props: PagePropType): JSX.Element {
           color="light"
           onClick={() => setModalOpen(true)}
         >
-          Add Import
+          Upload import data
```

Comment:
nit:  "Upload data for import" or "Upload files for import"

Reply:
Done